library/src/lib/toggle/toggle.component.ts

```diff
@@ -19,35 +23,50 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
     }
 })
 export class ToggleComponent implements OnInit, ControlValueAccessor {
+    /** @hidden */
     @ViewChild('input')
     inputElement: ElementRef<HTMLInputElement>;
 
+    /** @Input The size of the toggle. 
+     * Can be one of the four *xs*, *s*, *l*, *error* or default. 
+     */
     @Input()
     size: string;
 
+    /** @Input Whether the toggle is disabled. */
     @Input()
     disabled: boolean = false;
 
+    /** @Input Id for the toggle component. If omitted, a unique one is generated. */
     @Input()
     id: string;
 
+    /** @Input Whether the toggle is checked. */
     @Input()
     checked: boolean = false;
 
+    /** @Input aria-label attribute. */
     @Input()
     ariaLabel: string = null;
 
+    /** @Input aria-labelledby attribute. */
     @Input()
     ariaLabelledby: string = null;
 
+    /** @Output Event fired when the toggle is checked. */
```

Comment:
It is fired when the state of the toggle changes, whether it is checked or unchecked. Also explain that $event can be used to retrieve the new state of the toggle